app/views/admin/projects.html.erb

```diff
@@ -73,52 +48,107 @@ See doc/COPYRIGHT.rdoc for more details.
           <label for='name'><%= Project.model_name.human %>:</label>
           <%= text_field_tag 'name', params[:name] %>
       </li>
+      <li class="simple-filters--filter">
+        <label for='' aria-visible="false" style="visibility: hidden"><%= l(:button_apply) %></label>
```

Comment:
Ah, demo time makes for awful code...

I guess a `<span><!-- HACK remove me later --></span>` would not have done the trick, would it?

Just promise me that you have a list with things that you **will** (read: not should) fix after the demo and I will not have seen this hack :wink: